Ратков Максим

mattgroy opened the pull request:
[Посмотреть решение](https://hruview.herokuapp.com/mattgroy/markup-task-3/master/index.html)

mattgroy committed:
Update index.html

mattgroy committed:
Update index.css

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

nex2hex commented on index.css:
```diff
@@ -0,0 +1,241 @@
+*/
+
+body
+{
```
min-width

nex2hex commented on index.css:
```diff
@@ -0,0 +1,241 @@
+@import url('https://fonts.googleapis.com/css?family=Caveat|Kelly+Slab|Oswald|Playfair+Display|Russo+One&subset=cyrillic');
```
подключи один из шрифтов локально

nex2hex commented on index.css:
```diff
@@ -0,0 +1,241 @@
+
+.paper-article p
+{
+    /*даже так переносы не появляются*/
```
браузер не знает язык la
сделай на en или ru

nex2hex commented on index.css:
```diff
@@ -0,0 +1,241 @@
+
+.article-title-1
+{
+    font-family: 'Russo One', sans-serif;
```
можно записать одним свойством через font

nex2hex commented on index.css:
```diff
@@ -0,0 +1,241 @@
+    margin-right: 10px;
+    border: 1px solid currentColor;
+    float: left;
+    font-size: 1em;
```
`font: ...`

nex2hex commented on index.html:
```diff
@@ -3,7 +3,193 @@
+                </figcaption>
+            </figure>
+            <p lang="la">
+                Lorem ipsum dolor sit amet, consectetur adipisicing elit.
```
Текст лучше прогнать через https://www.artlebedev.ru/typograf/

nex2hex commented on index.html:
```diff
@@ -3,7 +3,193 @@
+            <figure>
+                <!-- <img src="https://tvoytrofey.ru/uploads/images/00/
+                    00/23/2014/01/17/c16908-big.jpg" alt="Сом Володя"> -->
+                <img src="https://hmp.me/cbqx" alt="Сом Володя">
```
указывай ширину и высоту у картинок, чтобы страница не прыгала в момент загрузки

nex2hex reviewed (changes requested):
🍅

mattgroy committed:
maybe will fail with value-list-comma-space-after

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

mattgroy committed:
la -> en (en-US)

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

mattgroy commented:
🔔 линтер ругается на длинные строки в css, если на несколько строк разделить, то ругается, что нет обрамляющего пробела после запятых, хотя в коммите они есть. Что делать в такой ситуации?

nex2hex commented:
1) поправь длину строки в https://github.com/urfu-2018/markup-task-3/blob/master/.htmllintrc

2)
См https://urfu-2018.github.io/slides/markup/03-fonts-and-text/index.pdf

```
@font-face {
    font-family: myCustomFont;
    src: url(myCustomFont.woff2) format(woff2),
    url(myCustomFont.woff) format(woff)
}
```

mattgroy committed:
comma hack

Правила для css линтера берутся из сторонней библиотеки хрюнделя, поэтому изменения .htmllintrc не имеют смысла. Хак с запятыми должен помочь, хоть я и понимаю, что по-хорошему так делать не стоит

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

nex2hex commented:
woff формата не хватает
